### PR TITLE
conflict resolver example.

### PR DESCRIPTION
**Pull request by jamiltz**
This change is [Reviewable](https://reviewable.io/reviews/couchbase/couchbase-lite-net/896)

### CODE REVIEW

File: docs/examples/api-walkthrough/api-walkthrough/ExampleConflictResolver.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿using System;
```
Comment (borrrden) [inline docs/examples/api-walkthrough/api-walkthrough/ExampleConflictResolver.cs]: This file needs a copyright header

File: docs/examples/api-walkthrough/api-walkthrough/Program.cs [hunk 2]
```diff
@@ -120,6 +122,23 @@ static void Main(string[] args)
...
+			 */
+            var theirs = new Document("buzz");
+            theirs.Set("status", "theirs");
+            var mine = new Document("buzz");
+            mine.Set("status", "mine");
+            database.Save(theirs);
+            database.Save(mine);
+
+            var conflictResolverResult = database.GetDocument("buzz");
+            Console.WriteLine($"conflictResolverResult doc.status ::: {conflictResolverResult.GetString("status")}");
```
Comment (borrrden) [inline docs/examples/api-walkthrough/api-walkthrough/Program.cs]: Consider pointing out that the document will have two revisions now

**Comment by jamiltz** [comment]
@borrrden added the copyright header and more description in the conflict resolver example.

**Comment by mr-jingles** [comment]
I have tried to reproduce this - the second Database.Save method will generate a second local revision. However since the two revisions are not conflicting with already existing aerver revisions this will not create a conflict. Both revisions will get pushed to the server, with the second revision as winning revisision. The _changes endpoint (with active_only=true&style=all_docs&include_docs=true') does not show two revisions for this document and the Conflict Resolver also does not get called. This is good as the document is not in a conflict state.

**Pull request merged**